region/maridia/outer/Crab Hole.json

```diff
@@ -386,7 +386,11 @@
           "canIBJ",
           {"and": [
             "SpaceJump",
-            "canMidAirMorph"
+            "canTrivialMidAirMorph",
+            {"or": [
+              "HiJump",
```

Comment:
This one feels fine. Maybe its the water+SJ, or the easier setup with the 2 wide entrance.

Reply:
Thinking about it a bit more, I think this one should probably keep `canMidAirMorph` too. It's not really easier than most of them, and it is a bit blind